active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,660 @@
+            Target="{x:Bind SaveButton}"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to.">
+        </muxc:TeachingTip>
+    </Button.Resources>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+   AutoSaveTip.IsOpen = true;
+}
+```
+
+Here's the result when this code runs and the user clicks the button.
+
+![A sample app with a teaching tip pointing at the save button. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipPointerSample.jpg)
+
+The IsOpen property also enables teaching tip to be implemented with data binding.
+
+XAML
+```XAML
+<Button x:Name="SaveButton" Content="Save" Click="OnFirstSaveButtonClick">
+    <Button.Resources>
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            IsOpen="{x:Bind ShowAutoSaveTip, Mode=OneWay}"
+            Target="{x:Bind SaveButton}"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to.">
+        </muxc:TeachingTip>
+    </Button.Resources>
+</Button>
+```
+
+C#
+```C#
+public bool ShowAutoSaveTip
+{
+    get { return (bool)GetValue(ShowAutoSaveTipProperty); }
+    set { SetValue(ShowAutoSaveTipProperty, value); }
+}
+ 
+public static readonly DependencyProperty ShowAutoSaveTipProperty = DependencyProperty.Register("ShowAutoSaveTip", typeof(bool), typeof(MainPage), new PropertyMetadata(false));
+
+
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+   ShowAutoSaveTip = true;
+}
+```
+
+### Non-pointing tips
+
+Not all tips relate to an element onscreen. For these scenarios, do not set the Target property and the teaching tip will instead display relative to the edges of the UWP window. However, a teaching tip can have the pointer removed while retaining placement relative to a UI element by setting the PointerMode property to "Off". 
+
+Unlike pointing teaching tips which can only be added to a resource dictionary, non-pointing teaching tips can also be added through the visual tree, as shown below.
+
+XAML
+```XAML
+<Button x:Name="SaveButton" Content="Save" Click="OnFirstSaveButtonClick">
+
+<muxc:TeachingTip x:Name="AutoSaveTip"
+    Title="Saving automatically"
+    Subtitle="We save your changes as you go - so you never have to.">
+</muxc:TeachingTip>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+   AutoSaveTip.IsOpen = true;
+}
+```
+
+![A sample app with a teaching tip in the bottom right corner. The tip title reads "Saving automatically" and the subtitle reads "We save your changes as you go - so you never have to." There is a close button on the top right corner of the teaching tip.](TeachingTipSampleApp.jpg)
+
+### Add an offset 
+
```

Comment:
In the picture above, I'm confused how the spacing happens without any offsets. Is there a default offset that's the size of the shadow or something?

Reply:
@jevansaks All this PM art needs to be replaced with actual screenshots of the control in use. I was just wanting on a version of the tip that has shadows re-enabled to do so.